cli_tools/gce_vm_image_import/importer/args.go

```diff
@@ -108,13 +111,25 @@ func (args *ImportArguments) ValidateAndPopulate(populator param.Populator,
 		return err
 	}
 
+	// Appending a namespace to the scratch bucket ensures that all logs and artifacts
+	// are contained in the same directory.
+	args.addNamespaceToScratchBucket()
 	if err := args.populateNetwork(); err != nil {
 		return err
 	}
 
 	return args.validate()
 }
 
+func (args *ImportArguments) addNamespaceToScratchBucket() {
```

Comment:
Using "Namespace" for  the import top level folder might be confusing, since Google storage docs talk about a single namespace for all buckets which this PR doesn't (and can't) address: https://cloud.google.com/storage/docs/naming-buckets

Maybe call it "ParentFolder", "TopLevelFolder" or something along those lines?

Reply:
Makes sense; I could see how someone skimming the linked doc might get confused and think these are related concepts.

Updated the helper method to refer to "path" instead of "bucket" and added some docs.